styles/base.css

```
@@ -4,6 +4,8 @@
 
 html,
 body {
+  scrollbar-gutter: stable;
+
   @apply size-full;
```

```suggestion
  @apply size-full
    [scrollbar-gutter:stable];
```

You can add it as a Tailwind class, there is no need to add it anywhere else 😄

---

this is interesting, didn't know this, thank you @canerakdas 😄✌